packages/components/src/filter-picker/index.js

```diff
@@ -128,11 +128,35 @@ class FilterPicker extends Component {
 		} ) );
 	}
 
+	getAllFilterParams() {
+		const { config } = this.props;
+		const params = [];
+		const getParam = ( filters ) => {
+			filters.forEach( ( filter ) => {
+				if (
+					filter.settings &&
+					! params.includes( filter.settings.param )
+				) {
+					params.push( filter.settings.param );
+				}
+				if ( filter.subFilters ) {
+					getParam( filter.subFilters );
```

Comment:
This is a little nitpicky, but the recursion here is unnecessary unless we support nested `subFilters`.

Reply:
Yup agreed. I built the `subFilter` system with recursion in mind to accomodate infinite sub filters. This was a premature additional feature because no designs ever came around with multiple levels of sub filtering. But, since we do support it, it is necessary here.